include/ord/Tech.h

```diff
@@ -54,6 +54,8 @@ class Tech
   void readLef(const std::string& file_name);
   void readLiberty(const std::string& file_name);
   odb::dbDatabase* getDB();
+  void set_thread_count(const int threads);
+  int thread_count();
```

Comment:
Since these are application level APIs I think they could be free functions in the ord namespace rather than Tech members, e.g.
```
ord::set_thread_count(40)
```